lib/ModelMixins/AutoRefreshingMixin.ts

```diff
@@ -30,20 +23,28 @@ export default function AutoRefreshingMixin<
     constructor(...args: any[]) {
       super(...args);
       // We should only poll when our map items have consumers
-      onBecomeObserved(this, "mapItems", this.startAutoRefresh.bind(this));
-      onBecomeUnobserved(this, "mapItems", this.stopAutoRefresh.bind(this));
+      reaction(
```

Comment:
One remote possibility is that the item gets added as an overlay instead of in the workbench. Then the `mapItems` check will hold, but the `inWorkbench` check fails.

Reply:
ahh yep that makes sense!
I'll revert it

Reply:
done